src/desktop/apps/article/components/InfiniteScrollNewsArticle.tsx

```diff
@@ -243,6 +258,8 @@ export class InfiniteScrollNewsArticle extends Component<Props, State> {
               // Only show rail if already rendering canvas
               showCollectionsRail={relatedArticles && showCollectionsRail}
               areHostedAdsEnabled={areHostedAdsEnabled}
+              shouldAdRender={renderAd}
+              articleSerial={i + 1}
```

Comment:
Update looks great, will want to pass `adPosition.index` into this spot too, since they represent the same thing (i think).